custom/icds_reports/tests/agg_tests/outputs/child_vaccines_2017-05-01.csv

```diff
@@ -0,0 +1,133 @@
+state_id,supervisor_id,child_health_case_id,month,due_list_date_1g_dpt_1,due_list_date_2g_dpt_2,due_list_date_3g_dpt_3,due_list_date_5g_dpt_booster,due_list_date_5g_dpt_booster1,due_list_date_7gdpt_booster_2,due_list_date_0g_hep_b_0,due_list_date_1g_hep_b_1,due_list_date_2g_hep_b_2,due_list_date_3g_hep_b_3,due_list_date_3g_ipv,due_list_date_4g_je_1,due_list_date_5g_je_2,due_list_date_5g_measles_booster,due_list_date_4g_measles,due_list_date_0g_opv_0,due_list_date_1g_opv_1,due_list_date_2g_opv_2,due_list_date_3g_opv_3,due_list_date_5g_opv_booster,due_list_date_1g_penta_1,due_list_date_2g_penta_2,due_list_date_3g_penta_3,due_list_date_1g_rv_1,due_list_date_2g_rv_2,due_list_date_3g_rv_3,due_list_date_4g_vit_a_1,due_list_date_5g_vit_a_2,due_list_date_6g_vit_a_3,due_list_date_6g_vit_a_4,due_list_date_6g_vit_a_5,due_list_date_6g_vit_a_6,due_list_date_6g_vit_a_7,due_list_date_6g_vit_a_8,due_list_date_7g_vit_a_9,due_list_date_anc_1,due_list_date_anc_2,due_list_date_anc_3,due_list_date_anc_4,due_list_date_tt_1,due_list_date_tt_2,due_list_date_tt_booster,due_list_date_1g_bcg
```

Comment:
Most of the CSV is empty and no date is there I think only a few has. Any reason why the test case had to go into aggregation test and not as a normal test with have like fetching data from table and comparing it? 
If not a major reason and since the table is pretty straightforward without rollups and having csv is not handling any special case. I think it would be much easier to maintain the test case as a normal test rather than a csv output compare.

Reply:
changed